package.json

```diff
@@ -17,7 +17,6 @@
     "@material-ui/icons": "^4.9.1",
     "history": "^5.0.0",
     "html2canvas": "^1.0.0-rc.7",
-    "jquery": "^3.5.1",
```

Comment:
Why change in package.json file?

Reply:
Removed jquery file since it was not needed any more.
Converted all jquery uses into vanilla Javascript.